content/en/docs/collector/internal-telemetry.md

```diff
@@ -34,15 +35,56 @@ service:
       address: '0.0.0.0:8888'
 ```
 
-You can enhance the metrics telemetry level using the `level` field. The
-following is a list of all possible values and their explanations.
+You can enhance the metrics telemetry using the `level` field. This field
+controls how verbose the metric is. The following is a list of all possible
+values and their explanations.
 
 - `none` indicates that no telemetry data should be collected.
 - `basic` is the recommended value and covers the basics of the service
   telemetry.
 - `normal` adds other indicators on top of basic.
 - `detailed` adds dimensions and views to the previous levels.
 
+By default, the Collector's internal metrics are distributed across verbosity
+levels, as shown here:
+
+{{< tabpane text=true >}} {{% tab "`basic`" %}}
```

Comment:
Thinking about this metric breakdown from a Collector user's perspective, I think it would make more sense to include the complete list emitted at each level, rather than the delta. I'd like to use the [docsy shortcode for reusing documentation](https://github.com/google/docsy/blob/712aa05897c1abe239786522131e83d69e5e1b4e/userguide/content/en/docs/adding-content/shortcodes/index.md#reuse-documentation) so we don't have to repeat the hard-coded lists here in each tab. @svrnm or @chalin, would you mind if I tried this out? It would involve creating a few extra files (for the lists) in the docs/collector directory.

Reply:
There are 2 tabs right now and we talk about duplicating the ones in "normal" to "detailed", is this right?

Reply:
Hi @svrnm.  There are three tabs: basic (yet to be filled in; working on that now), normal, and detailed. The basic list is shortest. The normal list includes the basic list, plus a few more. And the detailed list includes the normal list plus several more metrics. If we decide to keep the tabbed panes (see below), I can hard-code those lists, but there will be duplication. Or I could try the docsy shortcode, but I'd need to create separate files for the lists. How about I hard-code the lists for now, and we can decide later to change it if the Markdown is too cluttered?

Thanks, @chalin. The original ask was for a list of metrics by verbosity level (see Issue #4278). I checked with a user (just one, but they had a better viewpoint than me) how they would want to see this information presented, and they responded with two use cases: wanting to see all the metrics emitted at a given level and wanting to know at which level a specific metric is emitted. If we could column-sort the table, I think it could serve for both use cases, but I couldn't find a way to sort without introducing CSS and JavaScript (there's an [open docsy issue](https://github.com/google/docsy/issues/604) for this). In the absence of sorting, I think the tabbed panes are the best way to capture the first use case. But if you think the information is redundant, I'm quite happy to remove the tabs and refer to the table.

Reply:
Atm, I'd vote to drop these tabs, and direct the reader to the table towards the end of the page. I understand that this isn't ideal, but it's a good incremental step in the right direction to addressing #4278 and improving this page IMHO.

Then, I'd suggest commenting on https://github.com/google/docsy/issues/604 and upvoting the issue so that it is more likely to get worked on.

WDYT @svrnm et all?

If y'all vote to keep the tabs, I'm willing to go with the majority.

Reply:
Another possibility to consider:
We drop the tabs but re-sort the table first by Level and then alphabetically by metric name.

Reply:
> Another possibility to consider: We drop the tabs but re-sort the table first by Level and then alphabetically by metric name.

Yes! That was what I was going to suggest too. Actually, we could even split the big table into three separate tables, one for each level, and then drop the level column (as Severin suggested at some point I believe). WDYT?